tests/integration/istioio/trafficmanagement/trafficshifting/scripts/tcp_trafficshifting_test.txt

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# $snippet route_traffic_v1.sh
+$ kubectl apply -f @samples/tcp-echo/tcp-echo-all-v1.yaml@ -n istio-io-tcp-traffic-shifting
+# $endsnippet
+
+# $snippet check_tcp_echo_is_running.sh
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="tcp")].port}')
+# $snippet
+
+# $snippet send_traffic_to_echo_v1.sh
+$ for i in {1..10}; do \
+docker run -e INGESS_HOST=$INGRESS_HOST -e INGRESS_PORT=$INGRESS_PORT \
```

Comment:
typo

Reply:
Fixed the typo. There was one below as well. I fixed that one too.